colossalai/inference/sampler.py

```diff
@@ -1,6 +1,9 @@
 from typing import List, Tuple
```

Comment:
There exist some updates in sampling methods (addition of sampling methods) on dev branch. You might want to rebase or merge and then add those methods to sampler as well